IndyAV/simulation/indyav_gazebo/urdf/indyav_car.urdf.xacro

```diff
@@ -15,27 +15,33 @@
     </collision>
   </link>
 
-  <xacro:sylphase name="ins"
-                  xyz="0.05 0.0 0.05" rpy="0 0 0" />
+  <xacro:front_axle name="front_left_axle"  xyz="0.4 -1 -0.2"/>
+  <xacro:front_axle name="front_right_axle"  xyz="-0.4 -1 -0.2"/>
+
+  <xacro:front_wheel name="front_left_wheel" xyz="0 0 0" parent_link="front_left_axle_link"/>
+  <xacro:front_wheel name="front_right_wheel" xyz="0 0 0" parent_link="front_right_axle_link"/>
+  
+  <xacro:back_axle name="back_axle" xyz="0 1 -0.2"/>
 
-  <xacro:back_wheel name="back_left_wheel"  xyz="-1   0.4 -0.2"/>
-  <xacro:back_wheel name="back_right_wheel"  xyz="-1 -0.4 -0.2"/>
+  <xacro:back_wheel name="back_left_wheel"  xyz="0.4  0 0" parent_link="back_axle_link"/>
+  <xacro:back_wheel name="back_right_wheel"  xyz="-0.4 0 0" parent_link="back_axle_link"/>
+  
   <gazebo>
     <plugin name="back_wheel_plugin" filename="libbackwheel_plugin.so">
       <wheels>
-        <wheel_0> back_right_wheel_link </wheel_0>
-        <wheel_1> back_left_wheel_link </wheel_1>
+        <wheel_0> back_axle_link_to_back_right_wheel_link_joint </wheel_0>
+        <wheel_1> back_axle_link_to_back_left_wheel_link_joint </wheel_1>
       </wheels>
-      <max_thrust>1000</max_thrust>
+      <back_axle_joint> base_link_to_back_axle_link_joint </back_axle_joint>
+      <max_wheel_velocity>1000</max_wheel_velocity>
       <max_velocity>20</max_velocity>
       <topic_name> throttle </topic_name>
     </plugin>
   </gazebo>
-
-  <xacro:front_wheel name="front_left_wheel"  xyz="1 0.4 -0.2"/>
-  <xacro:front_wheel name="front_right_wheel"  xyz="1 -0.4 -0.2"/>
+  
+  <xacro:sylphase name="ins"
+                  xyz="0.0 0.05 0.05" rpy="0 0 0" />
```

Comment:
Hey while you are changin the kart params, do you mind making them the same as what is specified in this document put together by DanP? https://uflorida.sharepoint.com/:p:/r/teams/INDY/_layouts/15/guestaccess.aspx?e=BcgPvT&share=EdrDUqVw4EtGojp4K8Ul4k4BmIx1CSfJRxZJ_3OMzO7mZg

Reply:
What should I do about the different widths (front vs back)?

Reply:
What you did here is fine. Thanks!